app/routes/admin/content/translations.js

```diff
@@ -9,8 +9,20 @@ export default Route.extend({
     return [
       {
         code      : 'en',
-        sourceUrl : this.get('loader').load('/admin/content/translations/all')
+        sourceUrl : '#'
       }
     ];
+  },
+
+  actions: {
+    translationsDownload() {
+      this.set('isLoading', true);
+      this.get('loader')
+        .load('/admin/content/translations/all')
+        .catch(() => {
+          this.set('isLoading', false);
```

Comment:
it will always be loading after success if you set `isLoading` false only when it fails.

Reply:
yeah, jsut saw that. Initialised `isLoading` .

Reply:
No, what I meant is: you set `isLoading` to true when the download begins. You never set it to false when it is successfully over. It remains true in perpetuity.

Reply:
Oh right. I'll need to make it False `finally`.